src/services/me/index.js

```diff
@@ -33,6 +34,17 @@ class Service {
 			logger.warning(err);
 			throw new GeneralError('Can\'t find connected school.');
 		}
+		try {
+			const accounts = await this.app.service('/accounts').find({
+				query: {
+					userId,
+				},
+			});
+			user.externallyManaged = accounts.some((account) => !!account.systemId);
```

Comment:
You should know this by looking at `user.source` (preferred in the future) and `user.ldapDn` (sadly not yet refactored, but planned). Fake-SSO-Accounts (moodle, non-LDAP IServ) are basically normal users after the first login, iirc.

Reply:
If this changes overtime than let’s make a helper that can be adapted later, or?

Reply:
So I should also check user.source here or only user.source? It is also checked in the server at several places in a similar way by using systemId, I think

Reply:
+1 for the helper